faq.md

```diff
@@ -0,0 +1,145 @@
+  their system to make use of its multipath capability. Still, even when only
+  one network interface is available, MPTCP can be useful in mobility use-cases:
+  when often switching from one network to another without stopping the
+  connections. When servers don't support MPTCP, the connection continues in
+  "plain" TCP.
+
+- Servers usually don't directly benefit from MPTCP, because they are not moving,
+  and with fast and reliable connections. But their client will, and it will be
+  useful for the servers too: letting them switch from one network to another
+  without disconnection, not to restart a long operation again ; having faster
+  connections, not to hold a transfer for a too long time, etc. We recommend
+  enabling MPTCP on servers by default to let users choosing whether to use
+  MPTCP. When clients don't request to use MPTCP, server applications will
+  receive "plain" TCP sockets from the kernel when connections are `accept`ed,
+  making the performance impact minimal.
+</details> {: .ctsm}
+
+## Are there any performance impact to use MPTCP?
+MPTCP is engineered to improve network resilience and utilization without
+adversely affecting the performance of TCP applications. It adds a few bytes in
+each TCP packet, causing a manageable overhead (~1%) that becomes advantageous
+when leveraging multiple network paths, potentially increasing throughput and
+reliability.
+
+It is also important to note that, when clients don't request to use MPTCP,
+server applications will receive "plain" TCP sockets from the kernel when
+connections are `accept`ed, making the performance impact minimal.
+
+## Are there unsupported TCP socket options?
+MPTCP support most TCP extensions. It is possible some most obscure ones are not
+supported. If it is the case, please document your use-case in a new
+[issue](https://github.com/multipath-tcp/mptcp_net-next/issues/).
+
+For example, MPTCP in the Linux kernel is [currently](https://github.com/multipath-tcp/mptcp_net-next/issues/480)
+not compatible with KTLS yet.
+
+## What are the supported operating systems?
+<details markdown="block">
+<summary>MPTCP is supported in the official Linux kernel from version 5.6. Any
+applications can [easily use it](implementation.html). The adoption of MPTCP
+extends beyond to various platforms including macOS. But... </summary>
+
+The usage of MPTCP on macOS is somehow restricted:
+- It is easy only when applications use their
+  [SDK](https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/improving_network_reliability_using_multipath_tcp)
+- If not, it looks like applications need to use private libraries (we are not
+  even sure the headers are available) with specific functions to create sockets
+  that are apparently not documented, e.g.
+  [OpenSSH for macOS](https://github.com/apple-oss-distributions/OpenSSH/blob/main/openssh/sshconnect.c#L487).
+  (This might change in the future.)
```

Comment:
```suggestion
- There is some documentation for [connectx](https://opensource.apple.com/source/xnu/xnu-7195.81.3/bsd/man/man2/connectx.2.auto.html) and [disconnectx](https://opensource.apple.com/source/xnu/xnu-7195.81.3/bsd/man/man2/disconnectx.2.auto.html) system calls that mention multipath support, however it is not clear what is required to use these interfaces. There does not appear to be public example code.
```

Reply:
Might be better to speculate less here by focusing on the published information?